core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncoders.java

```diff
@@ -125,6 +135,9 @@ private static HttpEncodingType determineEncoding(String acceptEncoding) {
             if (!encodings.containsKey(HttpEncodingType.DEFLATE)) {
                 return HttpEncodingType.DEFLATE;
             }
+            if (!encodings.containsKey(HttpEncodingType.SNAPPY)) {
+                return HttpEncodingType.SNAPPY;
```

Comment:
I'm not 100% sure but I feel we should use `SNAPPY` before `DEFLATE`.
@line/dx Do you have any idea how we decided the order of compression algorithms?

Reply:
I guess the order should be in performance, and how widely used the algorithm is.
Since SNAPPY has reasonable performance but isn't supported in browsers, I think the current location after DEFLATE looks fine but others may have different opinions.

Reply:
That totally makes sense. I agree with you. 👍